src/main/java/Deadline.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDate;
+import java.time.format.TextStyle;
+import java.util.Locale;
+public class Deadline extends Task {
```

Comment:
Perhaps it would be better formatted if you keep a line between the imports and your class definition?